plugins/chat/app/controllers/chat/api/current_user_threads_controller.rb

```diff
@@ -3,26 +3,26 @@
 class Chat::Api::CurrentUserThreadsController < Chat::ApiController
   def index
     ::Chat::LookupUserThreads.call(service_params) do
-      on_success do
+      on_success do |threads:, tracking:, memberships:, load_more_url:, participants:|
         render_serialized(
           ::Chat::ThreadsView.new(
             user: guardian.user,
-            threads: result.threads,
-            channel: result.channel,
```

Comment:
`result.channel` is actually not defined and returns `nil`. `channel` is not needed anyway for this serializer.